### PR TITLE
LPS-204047 Fix modal form export entries layout

### PR DESCRIPTION
**Pull request by titokenzo**
https://liferay.atlassian.net/browse/LPS-204047

### CODE REVIEW

**Comment by titokenzo** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/export-form/ExportFormModal.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@
  * any later version.
  *
  * This library is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * ANY WARRANTY; without even te implied warranty of MERCHANTABILITY or FITNESS
```
Comment (cgmonte) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/export-form/ExportFormModal.tsx]: Even though you change the license text on next commit, this shouldn't be here, as if for any reason next commit gets dropped, this doesn't stay in master. Please, remove this change from this commit.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/export-form/ExportFormModal.tsx [hunk 2]
```diff
@@ -32,36 +32,41 @@ const ExportFormModal: React.FC<IProps> = ({
 				{Liferay.Language.get('export')}
 			</ClayModal.Header>
 
-			<form action={exportFormURL} method="post">
-				<ClayModal.Body>
+			<ClayModal.Body>
+				<form 
+					action={exportFormURL} 
+					id="exportFormEntriesFormId"
```
Comment (cgmonte) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/export-form/ExportFormModal.tsx:29]: I am no sure about this string. I have never seen a ID string ending in "Id", this is already implicit. You don't add "Id" suffixes to ID strings. On the bright side, down at the `<button>` `form` attribute, having "Id" kind of makes things clearer for beginners programmers who may not know that this is referencing a form ID. I would say: remove the suffix, make it only "exportFormEntriesForm". What do you think?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/export-form/ExportFormModal.tsx [hunk 3]
```diff
@@ -32,36 +32,41 @@ const ExportFormModal: React.FC<IProps> = ({
...
-								{Liferay.Language.get('ok')}
-							</ClayButton>
-						</ClayButton.Group>
-					}
-				/>
-			</form>
+						<ClayButton
+							displayType="primary"
+							onClick={onClose}
+							form="exportFormEntriesFormId"
```
Comment (cgmonte) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/export-form/ExportFormModal.tsx:53]: Read above comment. One should know that this is and Id.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/export-form/ExportFormModal.tsx [hunk 4]
```diff
@@ -1,15 +1,6 @@
...
- * the terms of the GNU Lesser General Public License as published by the Free
- * Software Foundation; either version 2.1 of the License, or (at your option)
- * any later version.
- *
- * This library is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even te implied warranty of MERCHANTABILITY or FITNESS
- * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
- * details.
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
```
Comment (cgmonte) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/export-form/ExportFormModal.tsx:3]: This is not the latest. Should it be? Se a 2023 license here:

https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/index.tsx#L1-L4

**Review by cgmonte — CHANGES_REQUESTED** [review]
Hi @titokenzo, thanks for sending this fix! Its was a very weird behavior and I have seen that in other places of the portal. It makes total sense to have the form inside the body and the button inside the footer, connected to the form by the `form` attribute. Nice fix. I made some minor change requests, please check them and let me know if anything does no make sense.

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by cgmonte** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**